index.html

```diff
@@ -56,12 +56,29 @@ <h1>New Entry</h1>
               <textarea id="notes" name="notes"></textarea>
             </div>
             <div class="column-full button-box">
-              <button type="submit" id="save-button" value="SAVE">SAVE</button>
+              <button
+                type="submit"
+                id="save-button"
+                value="SAVE"
+                class="form-button">
+                SAVE
+              </button>
             </div>
           </div>
         </form>
       </div>
-      <div data-view="entries" class="hidden"></div>
+      <div data-view="entries" class="hidden">
+        <div class="link-bar column-full">
+          <h1>Entries</h1>
+          <div class="entries-button-box">
+            <button class="entries-button">
+              <a>NEW</a>
+            </button>
```

Comment:
This shouldn't be a `button`, it should just be an `a` element styled to look like a `button`